client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./PreviousTripHashtagCard.css";
+import "../../App.css";
+
+// Function based React Component
+const PreviousTripHashtagCard = props => {
+  console.log(props);
+  // Default Class to apply to Component
+  const classList = `prev-hash-card-container`;
```

Comment:
If you're not going to change or update this class, then just add it as a string to the `className` property?